building/devices/nuvoton.rst

```diff
@@ -0,0 +1,61 @@
+.. _nuvoton:
+
+#######
+NUVOTON
+#######
+
+The instructions here will tell how to build and run OP-TEE for Nuvoton platform
+standalone (not as a part of openbmc image).
+
+
+.. _nuvoton_build_instructions:
+
+Build instructions
+******************
+
+=================
+Pre-requirements:
+=================
+
+#. Download and install the latest IGPS from the Nuvoton Israel github, i.e.,
+   $ git clone https://github.com/Nuvoton-Israel/igps-npcm8xx
+
+#. Download and extract the ARM GNU Toolchain 11.3 for aarch64
+   $ wget https://developer.arm.com/-/media/Files/downloads/gnu/11.3.rel1/binrel/arm-gnu-toolchain-11.3.rel1-x86_64-aarch64-none-elf.tar.xz?rev=73ff9780c12348b1b6772a1f54ab4bb3&hash=B80470312E67CF1F157D0A883FDEB279
+   $ tar xvf arm-gnu-toolchain-11.3.rel1-x86_64-aarch64-none-linux-gnu.tar.xz
```

Comment:
These instruction will become stale quite rapidly, is this version absolutely required? Can't a widely available toolchain be used, such as `aarch64-linux-gnu-gcc`, which is available on most Linux distributions with `sudo apt install gcc-aarch64-linux-gnu` or similar (and no need to update the user's PATH).

Reply:
According to Linaro requirements for TF-A, we are using aarch64-none-elf toolchain and not gcc-aarch64-linux-gnu. As for version, our instruction contains the current version we are using, we can update it to use the latest version. Do you want us to use different toolchains for TF-A and OPTee-OS?

Reply:
I have no strong opinion here. Historically, OP-TEE has always built fine with Linux toolchains and we're trying hard to preserve this possibility because it makes things simpler for users in general (one toolchain to build everything). Even if you use a `-none-` toolchain for your own builds, I believe it would be OK to simplify things in this documentation. But leave it as is if you prefer.